ci/pipelines/deploy-to-test.yml

```diff
@@ -2042,6 +2042,10 @@ jobs:
           file: tags/release-sha
         - load_var: candidate-image-tag
           file: tags/candidate-tag
+        - load_var: date
+          file: tags/date
+        - load_var: all-candidate-tags
```

Comment:
This var isn't being used later so you can remove the load of it from here.

otherwise this is great and feel free to update all the others :)